src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Host.sfxproj

```diff
@@ -2,7 +2,7 @@
   <Sdk Name="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <PropertyGroup>
-    <SkipBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</SkipBuild>
+    <SkipBuild Condition="'$(RuntimeFlavor)' != '$(PrimaryRuntimeFlavor)'">true</SkipBuild>
```

Comment:
Mono uses the apphost?

Reply:
On s390x we currently do not support CoreCLR, so we only have the Mono runtime.  We still would like to enable all .NET features (as far as technically possible), to make the resulting environment on s390x look as similar to the one on Linux on x64 as possible.

We currently cannot build the static "singlefilehost" image since its build process is closely tied to CoreCLR, but the dynamically linked "apphost" builds fine, even if it links to the Mono runtime.   This is why this patch enables the apphost, but skips the singlefilehost.

Reply:
I see -- I didn't realize that the apphost could be used to launch the Mono runtime